### PR TITLE
Fix initNameNode with hardcoded dns name

### PR DESCRIPTION
**Pull request by rmvz**

### CODE REVIEW

**Comment by elingg** [comment]
LGTM

File: hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java [hunk 1]
```diff
@@ -113,7 +113,8 @@ private void launchNameNodeTask(final ExecutorDriver driver, final TaskInfo task
     Runnable r = new Runnable() {
       public void run() {
         try {
```
Comment (elingg) [inline hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java]: nit: why was the indentation changed?

**Comment by elingg** [comment]
One minor nit, but should be good to merge!

**Comment by rmvz** [comment]
I have not looked closely at code formatting. Not sure if line break is correct, let me know if you prefer "+" sign at the end of the previous line, I have it at the start of the second line.

**Comment by elingg** [comment]
Thanks @rmvz. The line wrap is fine. We used to have an automated formatter prior to the gradle changes. It would be great to add it back in gradle to make this a bit easier! Merging now.

**Pull request merged**